examples/rc.tremby

```diff
@@ -23,15 +23,15 @@ bind ? help
 bind f filter
 bind F clear-filters
 
-set columns=artist year album discshort trackshort title length
-set sort=track disc album date albumartistsort
+set columns=artist originalyear year album discshort trackshort title length
+set sort=track disc album date originaldate albumartistsort
 set nonextafteraction
 
 set scrolloff=4
 
 set libraryroot=/home/bjn/media/music/
 
-set topbar=%ifcursong%%time_elapsed% %playstate% %time% (%progresspercentage%%%)%endif%\t%ifcursong%%artist%%endif%\tVol: %volume%%%  Mode: %muteshort%%consumeshort%%repeatshort%%randomshort%%singleshort%\n%listsize%\t%ifcursong%%album% (%year%)%endif%\tQ: %livequeuesize%\n\t%ifcursong%==> %title% <==%endif%
+set topbar=%ifcursong%%time_elapsed% %playstate% %time% (%progresspercentage%%%)%endif%\t%ifcursong%%artist%%endif%\tVol: %volume%%%  Mode: %muteshort%%consumeshort%%repeatshort%%randomshort%%singleshort%\n%listsize%\t%ifcursong%%album% (%originalyear% -> %year%)%endif%\tQ: %livequeuesize%\n\t%ifcursong%==> %title% <==%endif%
```

Comment:
Is it necessary to clutter the display like this? IMO it would be preferable to use only one of them. Maybe `originalyear`?